--- posteffect_radial_blur_frag.glsl	2021-05-15 13:57:33.248641500 -0600
+++ posteffect_2d_radial_blur_frag.glsl	2021-06-10 22:27:23.949015500 -0600
@@ -5,9 +5,7 @@
 ////////////////////////////////////////////////////////
 #version 120
 
-#extension GL_ARB_texture_rectangle : enable
-
-uniform sampler2DRect diffuseMap;
+uniform sampler2D diffuseMap;
 @define sampler_diffuseMap 0
 
 uniform float afSize;
@@ -33,8 +31,8 @@
 	@endif
 	vec2 vScreenCoord = gl_FragCoord.xy;
 	
-	vec2 vDir = avHalfScreenSize - vScreenCoord;
-	float fDist = length(vDir) / avHalfScreenSize.x;
+	vec2 vDir = vec2(0.5, 0.5) - vScreenCoord;
+	float fDist = length(vDir) / 0.5;
 	vDir = normalize(vDir);
 	
 	fDist = max(0.0, fDist-afBlurStartDist);
@@ -45,11 +43,11 @@
 	
 	for(int i=0; i<5; ++i)
 	{
-		vDiffuseColor += texture2DRect(diffuseMap, vScreenCoord+vDir*vSizeMul[i]).xyz * vColorMul[i];
+		vDiffuseColor += texture2D(diffuseMap, vScreenCoord+vDir*vSizeMul[i]).xyz * vColorMul[i];
 	}
 	
 	vDiffuseColor /= fTotalMul;
 	
 	gl_FragColor.xyz = vDiffuseColor;
 	gl_FragColor.w = 1.0;
-}
\ No newline at end of file
+}
